templates/locales/en.yml

```diff
@@ -734,6 +734,13 @@ en:
       downloader_interrupted: |-
         The download was interrupted by an external signal. It did not
         complete.
+      downloader_checksum_error: |-
+        The calculated checksum of the requested file does not match the expected
+        checksum!
```

Comment:
Could we include some information about the type of checksum in here (md5 vs sha256)?